packages/synapse-react-client/src/components/EntityCitation/EntityCitation.tsx

```diff
@@ -0,0 +1,170 @@
+
+  function getButtonText() {
+    if (isProjectAndEntityDoi) {
+      return 'Cite as...'
+    } else if (entityDoiAssociation && !projectDoiAssociation) {
+      return 'Cite page'
+    } else if (!entityDoiAssociation && projectDoiAssociation) {
+      return 'Cite project'
+    }
+    return ''
+  }
+
+  const buttonText = getButtonText()
+
+  const handleButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    if (entityDoiAssociation && !projectDoiAssociation) {
+      setSelectedDoi(entityDoiAssociation.doiUri)
+      setCitationAnchorEl(event.currentTarget)
+    } else if (!entityDoiAssociation && projectDoiAssociation) {
+      setSelectedDoi(projectDoiAssociation.doiUri)
+      setCitationAnchorEl(event.currentTarget)
+    } else {
+      // If both project and entity doi exist, show dropdown
+      setAnchorEl(event.currentTarget)
+    }
+  }
+
+  const popover = (
+    <CitationPopoverContent
+      doi={selectedDoi}
+      anchorEl={citationAnchorEl}
+      open={!!citationAnchorEl && !!selectedDoi}
+      id={id}
+      onClose={() => {
+        setCitationAnchorEl(null)
+        setSelectedDoi(undefined)
+      }}
+    />
+  )
+
+  const options: string[] = ['Cite this Project', 'Cite only this page']
+
+  if (!projectDoiAssociation && !entityDoiAssociation) {
+    return null
+  }
+
+  if (isEntityLoading || isProjectLoading) {
+    return (
+      <Skeleton
+        sx={{
+          width: { xs: '100%', sm: '140px' },
+          height: '45px',
+        }}
+      />
+    )
+  }
+
+  return (
+    <>
+      {isProjectAndEntityDoi ? (
+        <DropdownSelect
+          disabled={isEntityLoading && isProjectLoading}
+          options={options}
+          anchorRef={citationButtonRef}
+          sx={{
+            width: { xs: '100%', sm: 'initial' },
+          }}
+          buttonText="Cite as..."
+          variant="outlined"
+          buttonGroupAriaLabel="Citation options"
+          selectedIndex={selectedIndex}
+          setSelectedIndex={(index: number) => {
+            setSelectedIndex(index)
+            const selected =
+              index === 0
+                ? projectDoiAssociation?.doiUri
+                : entityDoiAssociation?.doiUri
+            if (selected) {
+              setSelectedDoi(selected)
+              setCitationAnchorEl(citationButtonRef.current)
+            }
+          }}
+        />
+      ) : (
+        <Button
+          onClick={handleButtonClick}
+          disabled={isEntityLoading && isProjectLoading}
+          sx={{
+            borderRadius: '3px',
+            padding: '6px 16px',
+            width: { xs: '100%', sm: 'initial' },
+          }}
+          variant="outlined"
+          startIcon={<DoubleQuotes width={18} height={18} />}
+        >
+          <Typography
+            variant="label"
+            sx={{ lineHeight: '20px', fontSize: '16px' }}
+          >
+            {buttonText}
+          </Typography>
```

Comment:
Sorry, to clarify, I don't think we need this special button styling at all--I think design would prefer that the text on this button has the same style as all of the other buttons--any specific styles that are different in the Figma for this feature were probably not intentional (but let me know if I am wrong!)

```suggestion
           {buttonText}
```

Reply:
So we don't want to use MUI Typography? To match the styling of the dropdown from the ActionMenu on synapse.org where this dropdown is placed next to, I should just do the styling in _core.scss in SWC?

Reply:
Removed styling, will add styling in synapse.org as needed.

Reply:
Shouldn't need to use Typography inside of a button. If the goal is style consistency with other items in a menu, then that styling should be co-located with the menu styles, not individual buttons.